app/javascript/mastodon/performance.js

```diff
@@ -2,9 +2,8 @@
 // Tools for performance debugging, only enabled in development mode.
 // Open up Chrome Dev Tools, then Timeline, then User Timing to see output.
 // Also see config/webpack/loaders/mark.js for the webpack loader marks.
-//
 
-let marky;
+import * as marky from 'marky';
```

Comment:
Before this was only conditinally required, as it's only needed in a specific debugging scope. Not sure if it can be done the same why with import

Reply:
It can not be done without changing the calls to Promises, but Webpack (or any bundler) should remove the calls as they are under a `process.env.NODE_ENV === "production"` guard, and then detect the `import` is not used, and not output it in production.

Reply:
It can not be done without changing the calls to Promises, but Webpack (or any bundler) should remove the calls as they are under a `process.env.NODE_ENV === "production"` guard, and then detect the `import` is not used, and not output it in production.

Reply:
OK, I guess the ESLint warning could be ignored then, and this dep moved back to devDeps then